manager/integration/tests/test_snapshot.py

```diff
@@ -187,6 +219,37 @@ def test_snapshot_hash_global_enabled_and_per_volume_enable_mode_without_immedia
                                   SNAPSHOT_DATA_INTEGRITY_DISABLED)
 
 
+def test_snapshot_hash_global_fast_check_and_per_volume_disable_and_with_immediate_hash(client, volume_name, settings_reset):  # NOQA
+    """
+    Check snapshots' checksums are not calculated
+    - global data-integrity is set to fast-check
+    - global immediate_hash is enabled
+    - per-volume data-integrity is set to disabled
+    """
+    prepare_settings_for_snapshot_test(client,
+                                       SNAPSHOT_DATA_INTEGRITY_FAST_CHECK,
+                                       "true",
+                                       "true")
+    check_per_volume_hash_disable(client, volume_name,
+                                  SNAPSHOT_DATA_INTEGRITY_DISABLED)
+
+
+def test_snapshot_hash_global_fast_check_and_per_volume_disable_and_without_immediate_hash(client, volume_name, settings_reset):  # NOQA
+    """
+    Check snapshots' checksums are not calculated
+    - global data-integrity is set to fast-check
+    - global immediate_hash is disabled
+    - per-volume data-integrity is set to disabled
+    """
+    prepare_settings_for_snapshot_test(client,
+                                       SNAPSHOT_DATA_INTEGRITY_FAST_CHECK,
+                                       "false",
+                                       "true")
+    check_per_volume_hash_disable(client, volume_name,
+                                  SNAPSHOT_DATA_INTEGRITY_DISABLED)
+
+
+@pytest.mark.long_test
```

Comment:
Hi @khushboo-rancher :
      I  introduce `long_test` marking for the tests which take more than 10 min
      
![Screenshot_20230105_163233](https://user-images.githubusercontent.com/104337648/210948574-9a2b6a26-b18a-428f-9b68-612594635532.png)

Reply:
@derekbit 
It's due to another ticket.[#5199](https://github.com/longhorn/longhorn/issues/5199#issuecomment-1371449299)
We need to introduce some additional marking for our test.

Reply:
Got it. Thank you.